app/src/main/java/com/simple/checkstyledemo/MainActivity.java

```diff
@@ -17,7 +16,9 @@ protected void onCreate(Bundle savedInstanceState) {
     }
 
     private void test(int a) {
-        if (a > 3) return;
-        System.out.println(a);
+        if (a > 3){
```

Comment:
**Reporter**: Checkstyle
**Rule**: com.puppycrawl.tools.checkstyle.checks.whitespace.WhitespaceAroundCheck
**Severity**: ERROR
**File**: app/src/main/java/com/simple/checkstyledemo/MainActivity.java L19

&apos;{&apos; is not preceded with whitespace.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>*